Update netservices.md and installation-guide (#542)
haiku/website: content/guides/daily-tasks/netservices.md

tags = []
+++

> The basic installation of Haiku contains several standard system network services including a FTP, Telnet, and SSH Server.

Reviewer: ```suggestion
The basic installation of Haiku contains several standard system network services including an FTP, Telnet, and SSH server.
```

Author: "a FTP", otherwise looks great.

Author: reuploaded revision

Reviewer: It actually is "an FTP server". With "a vs. an", it does not depend on how the following word is written (vowel -> an, consonant -> a), but how it's pronounced. "FTP" sounded out is "eff-tee-pee", starting with an "ee" sound, so it's "an FTP server".
Sorry to be such a nitpicker... :)

Author: I had no idea, thanks!

Haiku's default hostname is shredder. If shredder doesn't resolve, you can simply launch the
<span class='app'>Network</span> preference applet to find your system's network address.
You can change the hostname by editing /system/settings/network/hostname.</div>
<br>

<h3>ftpd - FTP server daemon</h3>
Haiku can easily be made to listen for incoming FTP connections to serve its files.

> <p>To start the FTP server select the Deskbar (the blue feather icon), Preferences, Network.  Select FTP Server and enable.</p>

Reviewer: It's actually a blue leaf. :)
IMO we can assume the users know there way around Haiku's most basic usage, i.e. the Deskbar and how to launch apps and preferences with it.

```suggestion
<p>To start the FTP server open the Network preferences, select **`FTP server`**  from its list of services and click **`Enable`**.</p>
```

The same goes for the telnet server below.

Haiku's default hostname is shredder. If shredder doesn't resolve, you can simply launch the
<span class='app'>Network</span> preference applet to find your system's network address.
You can change the hostname by editing /system/settings/network/hostname.</div>
<br>

<h3>ftpd - FTP server daemon</h3>
Haiku can easily be made to listen for incoming FTP connections to serve its files.

DISCLAIMER: ftpd does not support SFTP or FTPS so network traffic is unencrypted.

<p>To start the FTP server open the Network preferences, select **`FTP server`**  from its list of services and click **`Enable`**.</p>
<br>
<h3>telnetd - Telnet server daemon</h3>
Haiku can easily be made to listen for incoming telnet connections.

DISCLAIMER: telnet traffic is unencrypted.

<p>To start the Telnet server open the Network preferences, select **`Telnet server`**  from its list of services and click **`Enable`**.</p>
<br>
<h3>sshd - Secure shell daemon</h3>
SSH is a network protocol that allows for data to be exchanged using a secure channel between two networked devices. The most common use is shell access.

> The SSH server daemon in Haiku is already started on a deafult install, but it does not allow connections until configured.

Reviewer: ```suggestion
The SSH server daemon is set to start automatically on a default Haiku install, but it does not allow connections until configured.
```
